src/components/ConvictionCalculator.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import moment from 'moment';
+
+import ConvictionForm from './ConvictionForm';
+import ConvictionResults from './ConvictionResults';
+
+import EligibilityTimelineCalculator from '../logic/EligibilityTimelineCalculator';
+import { CalculatorInput, ConvictionInput } from '../logic/type/CalculatorInput';
+
+//import mockOutput from '../logic/mockdata/Output';
+
+const ConvictionCalculator = () => {
+  const calculator = new EligibilityTimelineCalculator();
+  const [hasResults, setHasResults] = React.useState(false);
+  const results = React.useRef();
+  const [convictions, setConvictions] = React.useState([]);
+
+  const addConvictions = () => {
```

Comment:
We need the input form the initially start with some number (say, 10) of rows so the user can start filling in the form without first clicking the "Add Conviction" button